sale_order_contact_attribute/readme/DESCRIPTION.md

```diff
@@ -1,5 +1,6 @@
 This module adds some contact-related fields to account.move.
```

Comment:
It is wrong, isn't? It should be `sale.order`.

Reply:
@AungKoKoLin1997 Thanks! Fixed it.